hack/helm_vars/wire-server/values.yaml.gotmpl

```diff
@@ -169,6 +170,7 @@ cannon:
     limits:
       memory: 512Mi
   drainTimeout: 0
+  disabledAPIVersions: []
```

Comment:
AFAICT, setting this for each service separately is the approach. This setting should be the same for all the services, though. It's mentioned in the `config-options.md` docs, but it might be good to add a note here as well.